sectionedrecyclerview/src/main/java/com/cruxlab/sectionedrecyclerview/lib/SectionDataManager.java

```diff
@@ -599,6 +613,23 @@ public void notifyRangeChanged(int section, int startPos, int cnt) {
             }
         }
 
+        @Override
+        public void notifyDataSetChanged(int section) {
+            short sectionType = sectionToType.get(section);
```

Comment:
Please add the following check to make sure section index is valid:
```suggestion
            checkSectionIndex(section);
            short sectionType = sectionToType.get(section);
```